### DISCUSSION TITLE
Cheapest source of WT32-ETH01 in 2024 ?

### DISCUSSION THREAD

**Discussion by shodanx2** [ldijkman/WT32-ETH01-LAN-8720-RJ45-#18] [General]
Hello,

Best I found so far is ali 1005006485369425 at 10.16$CAD

It used to be about 8.30$ but that was 3 years ago. 

Anyone knows of a better deal ? Even if that means buying 10 of them !

**Comment by ldijkman** [comment]
https://nl.aliexpress.com/item/1005006485369425.html

Looks good to me

Does it help to search for cheaper if cheaper shipping will be higher
